app/javascript/gobierto_visualizations/modules/contracts_controller.js

```diff
@@ -207,6 +208,25 @@ export class ContractsController {
     };
   }
 
+  _translateCategoriesTitle(contractsDataMap) {
+
+    const categoriesLanguage = I18n.locale === 'es' ? categoryTitlesEsp : categoryTitlesCat
+
+    contractsDataMap.map(d => {
+      const { category_title } = d
+      /*If the contract isn't categorized, we include it in other*/
+      if (!category_title) {
```

Comment:
We don't need this in the end, since they will never come as null (and if they do, we should know to fix it)

Reply:
Ok. Done!